doc/index.rst

```diff
@@ -263,24 +263,34 @@ ABI checks
 
 Generally speaking, the `ABI <https://en.wikipedia.org/wiki/Application_binary_interface>`_ of a library might break for various reasons. A detailed explanation and a list of DOs and DON'Ts can be found in the `KDE Community Wiki <https://community.kde.org/Policies/Binary_Compatibility_Issues_With_C%2B%2B>`_.
 
-The ABI checks with `industrial_ci` can be enabled by setting 'ABICHECK_URL' to a code archive with your stable version (e.g. tagged version or older tagged versions of your package)
+The ABI checks with `industrial_ci` can be enabled by setting 'ABICHECK_URL' to the stable version (i.e. you want to check against with this for sure) of your code. Two types of value can be passed to `ABICHECK_URL`:
+
+- Branch.
+
+  - Simplest thus recommended. Branch needs to be stable.
```

Comment:
This is what I would recommend by now (without experience!) to maintainers of repositories which **already** have stabe branches (=branches that mirror the released code).

Reply:
The simplest is to point it to your default branch. This way the push tests will be a NOOP, but for pull request it would test against the correct baseline.

But in general the user should point to the URL with an archive that contains the stable baseline version.
For example http://www.ros.org/reps/rep-0009.html demands ABI compliance for "even-cycle  patch releases" ffor >1.0.0 "stacks", but it is outdated.

I would suggest to keep ABI compatibility for all release of a certain distro, if possible.
Otherwise users might face some weird ABI-related bugs.